.github/workflows/pr-check.yml

```
@@ -60,3 +65,53 @@ jobs:
           cd ..
           cd frontend/src-tauri
           cargo fmt -- --check
+
+  # Frontend Test Job
+  frontend:
+    runs-on: ubuntu-latest
+    name: Frontend Tests
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)